libs/platform/experience/src/services/experience-data/experience-data.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Provider } from '@spryker-oryx/di';
+import { Component } from '@spryker-oryx/experience';
+
+export const ExperienceData = 'oryx.ExperienceData*';
+export const ExperienceDataService = 'oryx.ExperienceDataService';
+
+export const enum ExperienceDataMergeType {
+  Before = 'before',
+  After = 'after',
```

Comment:
We discussed append and prepend and the dynamic behaviour depending on the target (component with child components vs component without child components. I wonder if we should not allow for also adding before/after a composition. 
What about: before, after, append and prepend?

Append and prepend will only work with components that have child components though.